content/posts/introduction-to-dictionaries/index.mdx

```diff
@@ -160,13 +155,16 @@ The oldest dictionaries that we know of — those produced in the Middle East, A
     <p>
       16th tablet of the Urra=hubullu lexical series, Louvre Museum
 
-      <a
-        href="https://creativecommons.org/licenses/by/2.5"
-        target="_blank"
-      >
-        CC BY 2.5
-      </a>
-    </p>
+```
```

Comment:
why is this wrapped in a code block (including the closing `</p>`?

Reply:
i have no idea —  the PR was created by a colleague using the CMS, so i really don't know why this would appear here.